### ISSUE TITLE
Allow _source filtering fields on suggesters

### ISSUE THREAD

**Issue by ggalage** [barseghyanartur/django-elasticsearch-dsl-drf#109]
Hello thanks for great library.  
For building autocomplete search feature, it would be nice to have source filtering. https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-source-filtering.html
Is there a way to take advantage of this feature to minimize ES workload and bandwidth usage?

**Comment by barseghyanartur** [comment]
In general it's not recommended to use that approach. Why would you need that? Any advantages?

**Comment by ggalage** [comment]
All information I need when making suggestor completion query is the text I'm querying.  No need for other fields values. 

For example, if I'm searching on movies based on movie.titles, and i want to use suggester completion for autocomplete search widget, I want to return just the titles on _source using source filtering since thats all the info i need and especially when movie has lots of fields, it causes my es and server unnecessary load.

**Comment by barseghyanartur** [comment]
@ggalage:

I get it. Look, in that case, you should override the ``get_queryset`` method of your ViewSet in the following way:

```python

class MyViewSet(DocumentViewSet):
    # ...

    def get_queryset(self):
        qs = super(MyViewSet, self).get_queryset()  # Original queryset
        qs = qs.source(['title'])  # Queryset containing ``title`` field only.
        return qs  # Do not forget to return
```

That's it. You should do it for every ViewSet you need it for.

I would accept a pull request (with a ``SourceFilterBackend`` backend) which would add the functionality in a more dynamic way.

**Comment by barseghyanartur** [comment]
Solved in 0.17.4.
